packages/python-runner/src/widget.tsx

```diff
@@ -214,9 +214,24 @@ export class PythonFileEditor extends DocumentWidget<
 
     if (this.dockPanel.isEmpty) {
       // Add a tab to dockPanel
-      this.scrollingWidget = new ScrollingWidget({
+      const scrollingWidget = new ScrollingWidget({
         content: this.outputAreaWidget
       });
+      this.scrollingWidget = scrollingWidget;
```

Comment:
why separate these lines?

Reply:
L228 and L231 (the button onclick handlers) can't use this.scrollingWidget because this is different within that context.

Reply:
ah gotcha